### PR TITLE
APIGOV-28055 - remove outdated docs for aws agent deployments

### PR DESCRIPTION
**Pull request by jcollins-axway**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Cleanup no longer supported processing modes in AWS agents

## Deploy preview link

Updated the following pages to remove references to SQS and processing modes.

- http://pr-482.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_aws_gateway/
- http://pr-482.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_aws_gateway/deploy-your-agents-with-amplify-cli/
- http://pr-482.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_aws_gateway/cloud-administration-operation/
- http://pr-482.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_aws_gateway/deploy-your-agents-1/
- http://pr-482.amplify-central.opendocs-builder.pcloud.axway.int/docs/connect_manage_environ/connect_aws_gateway/tips-and-troubleshooting/

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 0b9ae50 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-482.amplify-central.opendocs-builder.pcloud.axway.int |

**Comment by jcollins-axway** [comment]
@vivekschauhan, @sbolosan, @alrosca, @dgghinea could you review the changes

File: content/en/docs/connect_manage_environ/connect_aws_gateway/_index.md [hunk 1]
```diff
@@ -21,21 +21,13 @@ Each AWS Gateway can be represented by an Amplify environment allowing you to be
...
 
 ### Traceability Agent
 
-The Traceability Agent is used to filter the AWS CloudWatch logs and prepare the transaction events that are sent to Amplify and visible in Business Insights. Viewing your traffic helps you to identify the bottleneck and errors. The traffic can be filtered by environment in case multiples are involved in your topography. As part of the deployment package, you can use cloud formation scripts to set up the following agent-dependent AWS Services:
+The Traceability Agent is used to filter the AWS CloudWatch logs and prepare the metric, usage and transaction events that are sent to Amplify and visible in Business Insights. Viewing your traffic helps you to identify the bottleneck and errors. The traffic can be filtered by environment in case multiples are involved in your topography. As part of the deployment package, you can use cloud formation scripts to set up the following agent-dependent AWS Services:
```
Comment (sbolosan) [inline content/en/docs/connect_manage_environ/connect_aws_gateway/_index.md:28]: ...helps you to identify possible bottlenecks and errors

File: content/en/docs/connect_manage_environ/connect_aws_gateway/_index.md [hunk 2]
```diff
@@ -21,21 +21,13 @@ Each AWS Gateway can be represented by an Amplify environment allowing you to be
...
 ### Discovery Agent
 
-The Discovery Agent is used to discover new deployments and stage updates to existing deployments for publishing related APIs in Amplify (either as a catalog item or as an API service). As part of the deployment package, use the provided cloud formation scripts to set up the following agent-dependent AWS Services:
-
-* AWS Config - Administers, audits and monitors resource configurations. Records and validates configuration changes.
-* AWS CloudWatch - Monitors resources and AWS applications in real time. Receives and routes supported AWS Service events.
-* AWS SQS - Decouples and scales microservices, distributed systems and serverless applications.
-
-![Service Discovery](/Images/central/connect-aws-gateway/aws-discovery-agent_v2.png)
+The Discovery Agent is used to discover new deployments and stage updates to existing deployments for publishing related APIs in Amplify as an API service. It will regularly check AWS API Gateway and check for modifications on these resources to push updates to Amplify.
```
[the remark this answers is not in the document]
Comment (jcollins-axway) [reply content/en/docs/connect_manage_environ/connect_aws_gateway/_index.md:24]: updated

**Review by sbolosan — APPROVED** [review]

**Review by alrosca — APPROVED** [review]

**Review by vivekschauhan — APPROVED** [review]

**Review by robertengaxway — APPROVED** [review]
I approve

**Pull request merged**